esi_leap/api/controllers/v1/node.py

```diff
@@ -47,6 +56,16 @@ def get_all(self):
         nodes = ironic.get_node_list(context)
 
         node_collection = NodeCollection()
-        node_collection.nodes = [Node(name=n.name) for n in nodes]
+
+        for node in nodes:
+            offers = offer_obj.Offer.get_all({"resource_uuid": node.uuid},
+                                             context)
```

Comment:
This will get all offers, including any expired ones - and since you only show the first one (which is reasonable), you'll probably end up showing an expired one. You may want to add more filters here to filter by status.

Reply:
Added a status filter